autogen.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+AUTOCONF=autoconf
```

Comment:
`: "${AUTOCONF:=autoconf}"` would allow to override the default.

Reply:
Good!

Reply:
> `: "${AUTOCONF:=autoconf}"` would allow to override the default.

Re-thinking to that. Because we can select to command to use with AUTOCONF_VERSION, using another variable is duplicated.

Reply:
On OpenBSD `AUTOCONF_VERSION` works and must be used if the user runs just `autoconf` (or other related commands), the reason why it works is [this script](https://github.com/openbsd/ports/blob/master/devel/metaauto/files/meta.in), which is a part of the `metaauto` OpenBSD package (`openbsd-amd64` would be a good example of what it looks like on the live filesystem).

On other systems the user always has to run the correct command. For example, this is Ubuntu 22.04:
```
$ autoconf
autoconf      autoconf2.69  
$ autoconf --version
autoconf (GNU Autoconf) 2.71
[...]
$ AUTOCONF_VERSION=2.69 autoconf --version
autoconf (GNU Autoconf) 2.71
$ autoconf2.69 --version
autoconf (GNU Autoconf) 2.69
```

Reply:
Thanks. Thus I will keep the two ways to choose:
```
AUTOCONF_VERSION=2.69 ./autogen.sh
AUTOCONF_VERSION=2.71 ./autogen.sh
```
and
```
AUTORECONF=autoreconf-2.69 ./autogen.sh
AUTORECONF=autoreconf-2.71 ./autogen.sh
```